plugins/database/cassandra/connection_producer.go

```diff
@@ -83,38 +82,46 @@ func (c *cassandraConnectionProducer) Initialize(ctx context.Context, req dbplug
 		return fmt.Errorf("username cannot be empty")
 	case len(c.Password) == 0:
 		return fmt.Errorf("password cannot be empty")
+	case len(c.PemJSON) > 0 && len(c.PemBundle) > 0:
+		return fmt.Errorf("cannot specify both pem_json and pem_bundle")
```

Comment:
I think that this is good to do, but note that we previously allowed both to be specified at the same time, with preference on `pem_json` so it's technically a breaking change. Would be good to get the docs updated as well to indicate that this is a `one of` between the two parameters.

Reply:
True, this is a breaking change. I'm not a fan of allowing for both as it can create confusion ("which one is being used?"). I'll update the docs to describe this change unless anyone has any objections to making this change.